discord/user.py

```diff
@@ -299,15 +299,22 @@ class ClientUser(BaseUser):
         Specifies if the user has MFA turned on and working.
     premium: :class:`bool`
         Specifies if the user is a premium user (e.g. has Discord Nitro).
+    premium_type: :class:`PremiumType`
+        Specifies the type of premium a user has (e.g. Nitro or Nitro Classic). Could be None if the user is not premium.
     """
-    __slots__ = ('email', 'verified', 'mfa_enabled', 'premium', '_relationships')
+    __slots__ = ('email', 'verified', 'mfa_enabled', 'premium', 'premium_type', '_relationships')
 
     def __init__(self, *, state, data):
         super().__init__(state=state, data=data)
         self.verified = data.get('verified', False)
         self.email = data.get('email')
         self.mfa_enabled = data.get('mfa_enabled', False)
         self.premium = data.get('premium', False)
+        premium_type = data.get('premium_type', None)
```

Comment:
```suggestion
        premium_type = try_enum(PremiumType, data.get('premium_type', None))
```

Make sure to import it and remove the previous lines. There's a reason why there's a helper for this.

Reply:
I wasn't aware of the helper, thanks Danny. Should be fixed in 6418454b26f64ab48df4dc69df0b47728de6cb0f.